packages/cdk-environment-resources/lib/index.ts

```diff
@@ -60,16 +61,38 @@ export class CdkEnvironmentResources extends cdk.Construct {
             groupMetrics: [autoscaling.GroupMetrics.all()]
         });
 
-        if(props.userData){
+        if (props.userData) {
             autoScalingGroup.addUserData(props.userData);
         }
 
         cdk.Tags.of(autoScalingGroup).add("EC2Group", "ecs-container-instance", {
             applyToLaunchedInstances: true
         });
 
-        const targetTrackingScalingPolicy = autoScalingGroup.scaleOnCpuUtilization("ScalingPolicy", {
-            targetUtilizationPercent: 50,
+        const memoryReservationMetric = new cloudwatch.Metric({
+            namespace: "AWS/ECS/ClusterName",
+            metricName: "MemoryReservation",
+            dimensionsMap: {
+                "ClusterName": `${props.stage}-${props.project}`
+            },
+            period: cdk.Duration.minutes(3)
+        });
+
+        const scalingPolicy = autoScalingGroup.scaleOnMetric("ScalingPolicy", {
+            metric: memoryReservationMetric,
+            scalingSteps: [
```

Comment:
It would be nice if we could parametrize these scaling steps since we will likely be doing some trial and error here.

Reply:
I will implement this following release.  It can be done manually I think for now.